crates/mast-package/src/package/manifest.rs

```diff
@@ -55,7 +68,7 @@ impl PackageManifest {
 
     /// Get the number of procedures exported from this package
     pub fn num_exports(&self) -> usize {
-        self.exports.len()
+        self.exports.values().filter(|export| export.is_procedure()).count()
     }
```

Comment:
Should we rename this to something like `num_proc_exports()` to indicate that this returns only the number of procedure exports?